src/Akka.Hosting/MauiSupport.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="MauiSupport.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2023 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2023 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Akka.Hosting
+{
+    public static class AkkaMauiSupport
+    {
+        public static async Task StartAkka(IServiceProvider provider, CancellationToken token = default)
```

Comment:
Akka hosted service have to be started manually by users, `AkkaHostedService` is internal, we have to use these wrapper methods to let user call the service `StartAsync()` method